api/app/Http/Controllers/ApiController.php

```diff
@@ -43,6 +47,20 @@ public function getAll()
         return $this->getResponder()->collection($this->getRepository()->all());
     }
 
+    public function getAllPaginated(PaginatedRequestDecoratorInterface $request)
+    {
+        $count = $this->getRepository()->count();
+        $limit = $request->getLimit(static::PAGINATOR_DEFAULT_LIMIT, static::PAGINATOR_MAX_LIMIT);
```

Comment:
yes, though of that
not sure if it is possible to test it inside controller, I will check that out